persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -96,26 +101,35 @@ public void execute() {
             errStream.println(commandUsageInfo);
             return;
         }
-
         try  {
             balProject = ProjectLoader.loadProject(Paths.get(sourcePath));
-            name = balProject.currentPackage().descriptor().org().value() + "." + balProject.currentPackage()
-                    .descriptor().name().value() + "." + "clients";
+            name = balProject.currentPackage().descriptor().name().value() + "." + "clients";
             setupJdbcDriver();
 
             balProject = BuildProject.load(Paths.get(sourcePath).toAbsolutePath());
             balProject.currentPackage().getCompilation();
-            configurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, this.configPath), name);
+            persistConfigurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, PERSIST,
+                    this.persistConfigPath), name);
+            String projectName = balProject.currentPackage().descriptor().name().value();
+
+            for (String key : persistConfigurations.keySet()) {
+                if (persistConfigurations.get(key).replaceAll("\"", "").equals(
+                        String.format(PLACE_HOLDER_TEMPLATE, projectName, key))) {
+                    populatePlaceHolders(persistConfigurations, name, projectName);
+                    break;
+                }
+            }
```

Comment:
Do we need to iterate here as we pass all configurations to the populate function

Reply:
In the most  recent update we only load the Config.toml if there are place holders in it. so we are breaking the iteration as soon as the configs are populated

Reply:
We need to separate the placeholder check and populate it. This logic is not sound good. Let's improve it later

Reply:
Sure